packages/server-wallet/src/models/__test__/fixtures/channel.ts

```diff
@@ -31,5 +31,10 @@ export const channelWithVars: Fixture<Channel> = fixture<Channel>(
   channel({vars: [addHash(stateSignedBy()())]})
 );
 
-export const withSupportedState = (stateVars: Partial<StateVariables>): Fixture<Channel> =>
-  fixture(channel({vars: [addHash(stateSignedBy(alice(), bob())({...stateVars}))]}));
+export const withSupportedState = (
+  stateVars: Partial<StateVariables>,
+  channelProps?: DeepPartial<RequiredColumns>
+): Fixture<Channel> =>
+  fixture(
+    channel({vars: [addHash(stateSignedBy(alice(), bob())({...stateVars}))], ...channelProps})
+  );
```

Comment:
I don't understand this change. `withSupportedState(someState)` already accepts `channelProps`, since its type is `Fixture<Channel>`.

Reply:
Oh I see. I didn't know how `Fixture` worked

Reply:
There is still an issue though because this fixture has computed properties computed by the defaults, e.g., `channelId`

Reply:
Yeah, the fixture pattern is "nice", but it is still rough around the edges. I think my suggestion does not work in this case.